akka-docs/rst/java/http/routing-dsl/directives/header-directives/optionalHeaderValue.rst

```diff
@@ -6,7 +6,7 @@ optionalHeaderValue
 Description
 -----------
 Traverses the list of request headers with the specified function and extracts the first value the function returns as
-``Some(value)``.
+``Optional[value]``.
```

Comment:
Ok, I used it because it is what the `toString` on optional returns (this and `Optional.empty` for empty ones)